docs/features/auth/azure-key-vault.md

```diff
@@ -1,59 +0,0 @@
----
-title: "Authentication for Azure Key Vault"
-layout: default
```

Comment:
We should add a redirect to our secret store page, or wherever the new content is to avoid having broken links

Reply:
Great minds think alike ☺️